dg/media/analytics/src/app/navs/navs.component.ts

```diff
@@ -0,0 +1,320 @@
+
+import { Component, OnInit, AfterViewInit, AfterViewChecked, Inject } from '@angular/core';
+import { DOCUMENT } from '@angular/platform-browser';
+import { GraphsService } from './navs.service';
+import { SharedService } from '../shared.service';
+import { global_filter } from '../app.component';
+import { GlobalFilterSharedService } from '../global-filter/global-filter-shared.service';
+import { DropDownItem } from './dropdown.model';
+import { config } from '../../config';
+
+@Component({
+  selector: 'app-navs',
+  templateUrl: './navs.component.html',
+  styleUrls: ['./navs.component.css']
+})
+
+export class NavsComponent implements OnInit,
+  AfterViewChecked {
+  private navClicked: boolean = false;
+  //used for collapse button
+  public isCollapsed: boolean = true;
+  public showOverall: boolean = true;
+  public showFilters: boolean = true;
+  public selectedNav: string = '';
+
+  //read config files from environment created for each app
+  private navsConfig = config.navsConfig;
+  private chartsConfig = config.chartsConfig;
+  private AddCommonOptions = config.AddCommonOptions;
+  generalConfig = config.generalConfig;
+  //keep track of nav switches and respective subnavs
+  toggleNav = {};
+  //dict with key as end nav and its corresponding containers
+  containers = {};
+  //list of charts in DOM
+  charts = [];
+  containerCharts = [];
+  filters = { 'params': {} };
+
+  constructor(private graphService: GraphsService, private _sharedService: SharedService,
+    private _globalfiltersharedService: GlobalFilterSharedService, @Inject(DOCUMENT) private document: any) {
+    this._sharedService.argsList$.subscribe(filters => {
+      Object.assign(filters.params, global_filter);
+      this.filters = filters;
+      Object.keys(this.containers).forEach(container => {
+        this.containers[container].applyFilter = false;
+      });
+      if (this.containers.hasOwnProperty('Home')) {
```

Comment:
mention this in config file i.e. "Home" attribute on dict. Maybe a variable inside defualt tab.